sunpy/map/header_helper.py

```diff
@@ -375,3 +376,70 @@ def _set_instrument_meta(meta_wcs, instrument, telescope, observatory, wavelengt
     'CD2_2':
     'Matrix element CDi_j describing the rotation required to align solar North with the top of the image.'
 }
+
+
+def make_heliographic_header(date, observer_coordinate, shape, *, frame, projection_code="CAR"):
+    """
+    Construct a FITS-WCS header for a heliographic (Carrington or Stonyhurst) coordinate frame.
+
+    The date-time and observer coordinate of the new coordinate frame
+    are taken from the input map. The resulting WCS covers the full surface
+    of the Sun, and has a reference coordinate at (0, 0) degrees Longitude/Latitude.
+
```

Comment:
Add a note saying that if more customization of the WCS is desired, use `make_fitswcs_header()` instead.

Reply:
I've added a See Also section for this.